app/javascript/src/components/Dashboard/Notes/NotesList.jsx

```diff
@@ -1,35 +1,103 @@
-import React, { useState } from "react";
+import React from "react";
+
+import EmptyNotesListImage from "images/EmptyNotesList";
+
+import EmptyState from "components/Common/EmptyState";
 
 import Note from "./Note";
-import EditNotePane from "./Pane/Edit";
 
-const NotesList = ({ notes, user, fetchNotes }) => {
-  const [showEditNote, setShowEditNote] = useState(false);
-  const [selectedNote, setSelectedNote] = useState({});
+const DUMMY_TAGS = [
+  "Gettings Started",
+  "Onboarding",
+  "User Flow",
+  "UX",
+  "Bugs",
+  "V2",
+];
+const STATUS = ["Drafted", "Created", "Updated"];
+const NAMES = [
+  "Debra",
+  "Allen",
+  "Gerald",
+  "Harris",
+  "Raymond",
+  "Carter",
+  "Jacqueline",
+  "Torres",
+];
 
-  const onEditClickHandler = selectedNote => {
-    setSelectedNote(selectedNote);
-    setShowEditNote(true);
+const NotesList = ({
+  notes,
+  onEditClick,
+  onDeleteClick,
+  setShowNewNotePane,
+}) => {
+  // Only for data creation purpose
```

Comment:
Please don't add unnecessary comments like these. These things ought be caught during self-review process. Read: https://handbook.neetokb.com/articles/git-and-github-essentials#self-review-process